lib/project_types/script/layers/domain/errors.rb

```diff
@@ -37,6 +37,8 @@ def initialize(extension_point_type, script_name)
         class MetadataNotFoundError < ScriptProjectError; end
 
         class MetadataValidationError < ScriptProjectError; end
+
+        class ServiceFailureError < ScriptProjectError; end
```

Comment:
I don't see where it's used?

Reply:
You are right, I must have added this when copying some code at some point and forgotten to remove it during a refactor.